certification/internal/utils/container/helper.go

```diff
@@ -66,3 +67,56 @@ func SaveContainerToFilesystem(podmanEngine cli.PodmanEngine, imageLog string) (
 	}
 	return tarpath, nil
 }
+
+type ContainerFn func(string) (bool, error)
+
+func unmountAndRemove(podmanEngine cli.PodmanEngine, containerId string) {
+	podmanEngine.Unmount(containerId)
+	podmanEngine.Remove(containerId)
+}
+
+func RunInsideContainerFS(podmanEngine cli.PodmanEngine, image string, containerFn ContainerFn) (bool, error) {
+	log.Debugf("PATH: %s", os.Getenv("PATH"))
+	createResult, err := podmanEngine.Create(image, &cli.PodmanCreateOptions{})
+	if err != nil {
+		log.Error("could not retrieve containerId", err)
+		return false, err
+	}
+	containerId := createResult.ContainerID
+
+	report, err := podmanEngine.Mount(containerId)
+	if err != nil {
+		log.Error("could not mount filesystem", err)
+		return false, err
+	}
+
+	defer unmountAndRemove(podmanEngine, containerId)
+
+	return containerFn(report.MountDir)
+}
+
+func GenerateBundleHash(podmanEngine cli.PodmanEngine, image string) error {
```

Comment:
Document this function, or rather - if this is not related, should we add this in a subsequent PR? I don't see it in use anywhere yet.

Reply:
Removed.